### PR TITLE
Update opensense example

### PR DESCRIPTION
**Pull request by aymanhab**
Use more reasonable names and shorter data files for opensense example and update setup files accordingly

### CODE REVIEW

**Comment by aymanhab** [comment]
@carmichaelong This part of IK for IMU fine tuning belongs to the models repo.

File: Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<OpenSimDocument Version="40000">
+	<IMUInverseKinematicsTool>
+		<!--Name of the directory where results are written. Be default this is the directory in which the setup file is be  executed.-->
```
Comment (carmichaelong) [inline Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml:4]: (noting here, but this is a core issue): Second sentence needs to be cleaned up. Should it be: "By default this is the directory from where the tool is run" or "By default this is the directory of the setup file"?

File: Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		<results_directory>C:\Users\Ayman-NMBL\Documents\OpenSim\4.2-2020-11-09-621c2af\Models\Rajagopal_OpenSense</results_directory>
```
Comment (carmichaelong) [inline Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml:5]: Probably don't want to hard code your directory here :)

**Review by carmichaelong — CHANGES_REQUESTED** [review]
Left 2 comments, and then one other note on the new .sto files: `placment_orientations.sto` and `walking_orientations.sto` contain the same data, which I don't think is what you intended?

File: Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml (hunk 1 above)
Comment (aymanhab) [reply Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml:4]: Agreed on the text update to "By default this is the directory of the setup file" in a core PR

**Comment by aymanhab** [comment]
> Left 2 comments, and then one other note on the new .sto files: `placment_orientations.sto` and `walking_orientations.sto` contain the same data, which I don't think is what you intended?

The contents are the same because placement needs only the first frame of data, we can cut the rest or use the same file but I prefer not to imply the connection between the two steps by reusing the same file. What do you think?

File: Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml (hunk 2 above)
Comment (aymanhab) [reply Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml]: Yes, removed.

**Comment by carmichaelong** [comment]
If you think it's best to keep the two files separate, then sounds good to me and cutting all extra lines from the placement file would be good.

**Comment by aymanhab** [comment]
Done, file trimmed.

**Review by carmichaelong — APPROVED** [review]

**Comment by aymanhab** [comment]
Thanks @carmichaelong

**Pull request merged**